mmv1/products/filestore/api.yaml

```diff
@@ -225,6 +225,19 @@ objects:
                 A list of IPv4 or IPv6 addresses.
               output: true
               item_type: Api::Type::String
+            - !ruby/object:Api::Type::Enum
+              name: 'connectMode'
+              description: |
+                The network connect mode of the Filestore instance.
+                If not provided, the connect mode defaults to
+                DIRECT_PEERING.
+              input: true
+              min_version: beta
+              default_value: :DIRECT_PEERING
```

Comment:
Setting a default here isn't safe, as the API doesn't return the value for preexisting instances. We'll want to remove this default and mark the field as `default_from_api: true` in `terraform.yaml`, I think.

Reply:
I removed default_value andI need to do some more reading on how to change that terraform.yaml and if that is needed. What's the impact of not having `default_from_api: true` in `terraform.yaml`?

Reply:
If the API returns a default value but the user hasn't set one, it'll cause a diff in Terraform.